templates/dedollarization/Trade.html

```diff
@@ -175,4 +175,7 @@
 </table>
 {% endif %}
 
+    <input type="hidden" name="trading"/>
```

Comment:
see comment above.
You also will need to remove the "some instructions here" text in the trade page, and translate the trade and results pages to spanish.

Reply:
Ok

Reply:
We are still doing the changes on the instructions, so maybe that's going to take a while

Reply:
k